### PR TITLE
AYR-1036 - Long TB name causing bad styling on Series Browse view

### PR DESCRIPTION
**Pull request by ltrasca**
## Changes in this PR
- removed the no-break class from the transferring_body and added a class for word-break
- fixed a small bug where the pagination buttons had a visited link color
- modified the width of transferring_body to be one quarter using the GDS override styles

## JIRA ticket
https://national-archives.atlassian.net/browse/AYR-1036

### Before
image

### After
image

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

**Comment by Terry-Price** [comment]
Looks good @ltrasca .
Does it work like this is the transferring body if multiple words in the same way. (no just one long word)???

**Comment by ltrasca** [comment]
> Looks good @ltrasca . Does it work like this is the transferring body if multiple words in the same way. (no just one long word)???

Hey @Terry-Price yes it works in the same way, I've attached a screenshot of an example below

image

**Review by colinbowen — APPROVED** [review]
Looks good, ok to merge once approved by Terry.

**Comment by Terry-Price** [comment]
All good then @ltrasca . thank you.

**Pull request merged**
